app/src/main/java/com/example/sopt_seminar/ui/FollowerFragment.kt

```diff
@@ -0,0 +1,90 @@
+package com.example.sopt_seminar.ui
+
+import android.os.Bundle
+import android.view.View
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.sopt_seminar.R
+import com.example.sopt_seminar.databinding.FollowerFragmentBinding
+import com.example.sopt_seminar.domain.model.Follower
+import com.example.sopt_seminar.ui.adapter.FollowerAdapter
+import com.example.sopt_seminar.util.BaseFragment
+import java.util.*
+
+class FollowerFragment : BaseFragment<FollowerFragmentBinding>(R.layout.follower_fragment) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, null)
+        val adapter = FollowerAdapter()
+
+        val simpleCallback = object : ItemTouchHelper.SimpleCallback(
+            ItemTouchHelper.UP or ItemTouchHelper.DOWN,
+            ItemTouchHelper.LEFT
+        ) {
+            override fun onMove(
+                recyclerView: RecyclerView,
+                viewHolder: RecyclerView.ViewHolder,
+                target: RecyclerView.ViewHolder
+            ): Boolean {
+                val startPosition = viewHolder.adapterPosition
+                val endPosition = target.adapterPosition
+                adapter.moveItem(startPosition, endPosition)
+                Collections.swap(testList, startPosition, endPosition)
+                return true
+            }
+
+            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
+                val position = viewHolder.adapterPosition
+                when (direction) {
+                    ItemTouchHelper.LEFT -> {
+                        adapter.removeItem(position)
+                        testList.removeAt(position)
+                    }
+                }
+            }
+        }
+
+        with(binding) {
+            followerRecyclerView.adapter = adapter
+            followerRecyclerView.layoutManager = LinearLayoutManager(context)
+            ItemTouchHelper(simpleCallback).attachToRecyclerView(followerRecyclerView)
+            adapter.submitList(testList.toCollection(mutableListOf()))
+            adapter.setOnItemClickListener(object : FollowerAdapter.OnItemClickListener {
+                override fun onItemClick(position: Int) {
+                    val action = HomeFragmentDirections.actionHomeFragmentToDetailFragment(
+                        followerName = testList[position].name,
+                        followerDes = testList[position].description
+                    )
+                    findNavController().navigate(action)
+                }
+            })
+        }
+    }
+
+    companion object {
+        private val testList: MutableList<Follower> = mutableListOf(
```

Comment:
```suggestion
        private val testList = mutableListOf(
```
코틀린에서는 타입추론 기능이 있기때문에 이렇게만 표현하셔도 됩니다!

Reply:
수정했습니다!!